### ISSUE TITLE
Error while loading networkX graph into DGL

### ISSUE THREAD

**Issue by lmegha** [dmlc/dgl#676]
I get this error when I try to convert a networkx graph (created from neo4j Data) into DGL graph,

**DGLError: [19:21:15] /opt/dgl/src/graph/graph.cc:53: Check failed: HasVertex(src) && HasVertex(dst) Invalid vertices: src=8192 dst=402945**

Can you explain why this happens? I suspected it might be because the edge didn't exist between those two nodes in the networkx graph but it does.

Any guidance on fixing this will be very helpful. Thanks!

**Comment by yzh119** [comment]
Hi, you may refer to [this thread](https://discuss.dgl.ai/t/graph-loading-error-from-networkx/309) in our discussion forum.

**Comment by lmegha** [comment]
Hi, So that thread definitely helps. Thank you for pointing me to it. I although still can't get it to work even though I mapped my nodeIds manually to the range(no_of_vertices) and my relationship tuples accordingly. 

Briefly my nodeId list looks like this [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11.....99987] and my relationship tuples look like this
(99974, 1)
(99858, 1)
(99986, 2)
(99944, 2)
(99983, 3)
(99467, 3)
(99986, 4)
(99837, 4)
(99986, 5)
(99888, 5)
(99987, 6)
(99957, 6)
(99983, 7)

I get the following error: DGLError: [18:27:05] /opt/dgl/src/graph/graph.cc:53: Check failed: HasVertex(src) && HasVertex(dst) Invalid vertices: src=99987 dst=6

My graph is a Directed Self looping graph, the code I used to create it is as follows:
`import networkx as nx
G = nx.MultiDiGraph()
G.add_nodes_from(nodeIds)
G.add_edges_from(edges)
import dgl
dgl = dgl.DGLGraph(G)`

The information of my networkX graph is as follows:
Name: 
Type: DiGraph
Number of nodes: 99987
Number of edges: 199257
Average in degree:   1.9928
Average out degree:   1.9928

Any help to guide me what I am doing wrong will be helpful! Thanks.
